src/main/java/com/codegym/shopyy/model/CartItem.java

```diff
@@ -0,0 +1,39 @@
+package com.codegym.shopyy.model;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.persistence.*;
```

Comment:
use import-specific library instead of import .*

Reply:
resoled